_includes/song.vto

```diff
@@ -1,34 +1,59 @@
 ---
 layout: layouts/page.vto
 ---
-Creators: {{tags}}
 
-{{ if composer}}
-Composer: {{composer}}
-{{ /if }}
+<div class="post-credits">
+  <div class="left-credits">
+      {{ if lyricist}}
+      <div>Lyricist: {{lyricist}}</div>
+      {{ /if }}
+  </div>
 
-{{ if lyricist}}
-Lyricist: {{lyricist}}
-{{ /if }}
+  <div class="right-credits">
+    <div>Creators: {{tags}}</div>
 
-{{ if arranger}}
-Arranger: {{arranger}}
-{{ /if }}
+    {{ if composer}}
+    <div>Composer: {{composer}}</div>
+    {{ /if }}
 
+    {{ if arranger}}
+    <div>Arranger: {{arranger}}</div>
+    {{ /if }}
+  </div>
+</div>
 
-<div id="osmdContainer"></div>
+<div id="osmdContainer" ></div>
 <script src="https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/releases/download/1.8.8/opensheetmusicdisplay.min.js"></script>
+
 <script>
-const osmd = new opensheetmusicdisplay.OpenSheetMusicDisplay("osmdContainer");
-osmd.setOptions({
-    backend: "svg",
-    drawTitle: false,
-    drawCredits: false,
-    scale: 0.8
-});
-
-osmd.load(`{{content}}`).then(function() {
+  function getScaleForScreenWidth() {
+    const screenWidth = window.innerWidth;
+    if (screenWidth > 2400) {
+      return 2
+    } else {
+      return 0.6 + (screenWidth - 600) * (1.4 / 1800);
+    }
+  }
```

Comment:
This doesn't scale well for desktop. I should be able to view a large part of the score.
![image](https://github.com/tskarhed/scoresly/assets/1438972/786feda2-96db-4fb0-8d45-082dd6f50442)